lib/full-site-editing/class-wp-rest-templates-controller.php

```diff
@@ -196,6 +196,16 @@ public function update_item( $request ) {
 			return new WP_Error( 'rest_template_not_found', __( 'No templates exist with that id.', 'gutenberg' ), array( 'status' => 404 ) );
 		}
 
+		// If the request explicitly sets `is_custom` to false,
+		// delete the template post, and return the template file.
+		if ( isset( $request['is_custom'] ) && false === $request['is_custom'] ) {
```

Comment:
I wonder if we should check that it's the only updated property. Because others will be ignored anyway.

Reply:
Ah, yeah I was [wondering about that](https://github.com/WordPress/gutenberg/pull/28141#issuecomment-761011020) 🙂 
Your reasoning makes sense!